client/src/components/Input.js

```diff
@@ -0,0 +1,31 @@
+import styled from '@emotion/styled';
+
+const Input = styled.input`
+  outline: none;
+  cursor: pointer;
+`;
+
+export const LoginInput = styled(Input)`
+  text-align: center;
+  width: 274px;
+  height: 60px;
+  background: white;
+  color: white;
+  border: 1px #707070 solid;
+  border-radius: 30px;
+  box-shadow: 0px 3px 6px grey;
+  font-size: 22px;
+  padding: 0px 0px;
+  outline: none;
+  cursor: pointer;
+  ::-webkit-input-placeholder {
+    text-align: center;
+    color: #b7b7b7;
+  }
+  &:focus {
+    background-color: #ffffff;
+    color: #0e0f09;
+  }
+`;
+
+export default Input;
```

Comment:
You have two exports here. A default export `Input` and a named export `LoginInput`. 
I like the idea to have a base Input, but is it required for you? What makes the `LoginInput` special, that it is only for the `Login`? 
I would reduce this file to one component `Input` only and combine the styles. 
If you do that, you can directly import the default in your stories:
`import Input from '../components/Input';`